spec/system/visit_invite_user_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'InviteToTeam', type: :system do
+  before(:each) do
+    login_gds_user
+  end
+
+  it 'shows form' do
+    visit invite_to_team_path(Team.first.id)
+    expect(page).to have_content 'Invite a new user'
+  end
+
+  it 'submit empty form shows errors' do
```

Comment:
These should read like an English sentence starting with "It", so for example
```
it 'shows errors when an empty form is submitted' do
```

Reply:
And I like your suggestion here so I have accepted it.